src/objects/Player.cpp

```diff
@@ -42,18 +42,13 @@ void Player::UpdateCameraPosition(bool animated) {
 }
 
 void Player::updateCameraPitch() {
-    float hypotenus = sqrt(
-        pow(cameraDistance, 2) +
-        pow(cameraHeight, 2)
-    );
-    float cosine = cameraDistance / hypotenus;
-    float angleInRadians = std::acos(cosine);
+    float angleInRadians = std::atan(cameraHeight /  cameraDistance);
     float angleInDegrees = angleInRadians * (180.0 / M_PI);
-    camera.UpdatePitch(angleInDegrees * -1.0f);
+    camera.UpdatePitch(angleInDegrees);
 }
 
 void Player::UpdateCameraYaw(bool animated) {
-    double yaw = -GetRotationYAngle() + 90.0f;
+    double yaw = 180.0 - GetRotationYAngle();
```

Comment:
As soon as I "fixed this" some weeks ago, I already forgot why I did it like that. That is a good reminder for me to leave more comments, especially in computations like this - where intention is not clear from the very first look.

I just remember I wanted to have always positive Yaw values to simplify debugging.